src/sindy.jl

```diff
@@ -1,18 +1,25 @@
+# Build up an enum
+@enum SparseRegressionAlg strridge sparseconvex
```

Comment:
ridge regression won't give sparsity

Reply:
Simply rename? I think until SR3 this is our best shot. Both work fine given a sufficient data quality.

Reply:
no, instead of a ridge it should really be a lasso. A lasso (L1) is sparsifying, ridge (L2) is not. https://stats.stackexchange.com/questions/45643/why-l1-norm-for-sparse-models